en/drafts/originals/image-classification-neural-networks.md

```diff
@@ -25,150 +25,154 @@ doi: tbc
 
 # Understanding Artificial Neurons
 
-How exactly do artificial neurons work? Rather than diving directly into training them, it is helpful to gain a broader understanding of their infrastructure. Let's say we are interested in a simple task such as determining if an image is a picture of a square or triangle. If you have done any kind of coding, you will know that most programs solve this by processing a sequence of steps. Loops and statements (such as `for`, `while`, `if`, etc.) allow our program to have branches that simulate logical thinking, and in the case of determining if an image contains a shape, we could code our program to count the number of sides and display "square" if it finds four or "triangle" if it finds three. Distinguishing between geometric objects may seem like a relatively simple task, but it requires a programmer to not only define a shape's characteristics but also implement logic that discerns those characteristics. This becomes increasingly more difficult as we run into scenarios where distinctions between images are more complex. For instance, look at the following images:
+How exactly do artificial neurons work? Rather than diving directly into training them, it is helpful to first gain a broader understanding of their infrastructure. Let's say we are interested in a simple task such as determining if an image is a picture of a square or triangle. If you have done any kind of coding, you will know that most programs solve this by processing a sequence of steps. Loops and statements (such as `for`, `while` and `if`) allow our program to have branches that simulate logical thinking. In the case of determining whether an image contains a shape, we could code our program to count the number of sides and display "square" if it finds four, or "triangle" if it finds three. Distinguishing between geometric objects may seem like a relatively simple task, but it requires a programmer to not only define a shape's characteristics but also implement logic that discerns those characteristics. This becomes increasingly difficult as we run into scenarios where distinctions between images are more complex. For instance, look at the following images:
 
-{% include figure.html filename="cat.jpg" caption="Figure 1. A picture of a cat" %}
-{% include figure.html filename="dog.jpg" caption="Figure 2. A picture of a dog" %}
+{% include figure.html filename="cat.jpg" alt="Visual description of figure image" caption="Figure 1. A picture of a cat" %}
```

Comment:
As part of our commitment to accessibility, we are now asking authors to provide descriptive alt-text to accompany figure images.

Please slot the alt-text into the template I've created throughout the lesson: `alt="Visual description of figure image"`.

I've collated [some resources on our Wiki](https://github.com/programminghistorian/jekyll/wiki/Achieving-Accessibility-Alt-text-Colour-Contrast#alt-text) as guidance if you need it.

Reply:
Will do.